models.py

```diff
@@ -231,12 +118,9 @@ def get_comms_name(self):
 	def get_engi_name(self):
 		engi_member = Member.query(Member.id == self.engi).get()
 		return engi_member.name
-	
+
```

Comment:
What changed here?

Reply:
Vim is being dumb. Will fix once I get it working functionally